Examples/CaseStudiesTests/PresentationTests.swift

```diff
@@ -300,6 +300,64 @@ final class PresentationTests: XCTestCase {
     }
     await assertEventuallyEqual(nav.viewControllers.count, 2)
   }
+
+  @MainActor
+  func testDismissMultiplePresentations() async throws {
+    class VC: UIViewController {
+      @UIBinding var isPresented = false
+      override func viewDidLoad() {
+        super.viewDidLoad()
+        present(isPresented: $isPresented) { VC() }
+      }
+    }
+
+    let vc = VC()
+    try await setUp(controller: vc)
+
+    vc.isPresented = true
+    await assertEventuallyNotNil(vc.presentedViewController)
+
+    try XCTUnwrap(vc.presentedViewController as? VC).isPresented = true
+    await assertEventuallyNotNil(vc.presentedViewController?.presentedViewController)
+
+    vc.isPresented = false
+    await assertEventuallyNil(vc.presentedViewController, timeout: 2)
```

Comment:
this test would fail because doing `vc.isPresented = false` would ultimately call `dismiss` on the child controller, which dismissed _its_ presentation rather than dismissing itself.